server/routes/sightingsController.js

```diff
@@ -0,0 +1,38 @@
+// const mongoose = require('mongoose');
+
+const Sighting = require('../models/Sighting.js');
+
+// Get all sightings in the database
+exports.getSightings = function (req, res) {
```

Comment:
You dont want to be passing `req`/`res` around. You want to make your functions as simple as possible, "separation of concerns".

Keep the req/res in the api route, and call `getSightings` and respond based on its return value. Then functions you're using can have tests written for them (though direct db call functions don't need this), and reused